strategies/FeaturePatienceStrategy.py

```diff
@@ -10,14 +10,16 @@ class FeaturePatienceStrategy(PolynomialAdaptivePatienceStrategy):
     Patience equation is influenced by features of the dataset currently being "trained" on.
     """
 
-    _name = "feature_patience"
-    _short_name = "FP"
+    _name = "feature_patience_v2"
```

Comment:
can you change these names so they no longer have v2?

Reply:
thanks, missed that one, updated